routes/apiuserroutes.js

```diff
@@ -84,11 +83,11 @@ router.patch("/",async (req,res) => {
         }
     }
     console.log(queryUpdate)
-    console.log(q)
+    // console.log(q)
     try{
-        const updateQuery= await User.updateOne(q, queryUpdate, { runValidators: true, context: 'query'} )
+        const updateQuery= await User.updateOne({"_id" : `${userid}`}, queryUpdate, { runValidators: true, context: 'query'} )
         res.json({"status" : updateQuery.ok, "data" : await User.findOne(queryUpdate)})
```

Comment:
#### Problem
```javascript
 res.json({"status" : updateQuery.ok, "data" : await User.findOne(queryUpdate)})
```
```queryUpdate``` will not show the correct data because ```queryupdate``` contains the ```$set``` changes

#### FIX

```javascript
 res.json({"status" : updateQuery.ok, "data" : await User.findbyId(userid)})
```